src/Algebra/Graph.hs

```diff
@@ -1010,6 +1010,22 @@ induce p = foldg Empty (\x -> if p x then Vertex x else Empty) (k Overlay) (k Co
     k f x     y     = f x y
 {-# INLINE [1] induce #-}
 
+-- | Construct the /induced subgraph/ of a given graph by removing the 
+-- vertices that are Nothing.
+-- Complexity: /O(s)/ time, memory and size.
+-- @
+-- induceJust ('vertex' (Nothing :: Maybe Int))             == 'empty'
```

Comment:
I suggest to omit the type signature in this example, to make it a bit easier to read.